docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Install and upgrade issues on K8s
+headerTitle: Install and upgrade issues on K8s
+linkTitle: Install and upgrade issues
+description: Troubleshoot issues encountered when installing or upgrading YugabyteDB Anywhere on K8s.
+menu:
+  preview_yugabyte-platform:
+    identifier: install-upgrade-kubernetes-issues
+    parent: troubleshoot-yp
+    weight: 10
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li>
+    <a href="../vm/" class="nav-link">
+      <i class="fa-solid fa-building"></i>
+      VM
+    </a>
+  </li>
+
+  <li>
+    <a href="../kubernetes/" class="nav-link">
+      <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+You might encounter issues during installation and upgrade of YugabyteDB Anywhere.
+
+If you experience difficulties while troubleshooting, contact [Yugabyte Support](https://support.yugabyte.com).
+
+## Unable to schedule a pod once
+
+There can be multiple reasons behind the failure of pod scheduling for YBA pods. The following can be the reasons behind it.
```

Comment:
Let's link to https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/#kube-scheduler-implementation